File: lab-2/src/pokemons/Virizion.java
package pokemons;

import moves.*;
import ru.ifmo.se.pokemon.*;

public class Virizion extends Pokemon{
    public Virizion(String name, int level) {
        super(name, level);
        setStats(91,90,72,90,129,108);
        setType(Type.GRASS, Type.FIGHTING);
        setMove(new Blizzard(), new SteelWing(), new AquaJet(), new RockSlide());
    }
}

File: lab-2-kotlin/src/pokemons/Virizion.kt
package pokemons

import moves.*

import ru.ifmo.se.pokemon.*

class Virizion(name: String?, level: Int) : Pokemon(name, level) {
    init {
        setStats(91.0, 90.0, 72.0, 90.0, 129.0, 108.0)
        setType(Type.GRASS, Type.FIGHTING)
        setMove(Blizzard(), SteelWing(), AquaJet(), RockSlide())
    }
}